Composer/packages/electron-server/src/appUpdater.ts

```diff
@@ -108,6 +111,7 @@ export class AppUpdater extends EventEmitter {
 
   private onDownloadProgress(progress: any) {
     log('Got update progress: %O', progress);
+    this.downloadingUpdate = true;
```

Comment:
Auto update won't call `public downloadUpdate()`,  So seems here is the only place we can know current is downloading.